local-spa/extra-styles.css

```
@@ -0,0 +1,384 @@
+		var(--local-exp-current-surface) var(--local-exp-input-lift-surface-weight),
+		white var(--local-exp-input-lift-white-weight)
+	) !important;
+}
+
+/* Phase B - Carbon layering PoC */
+body.sky-theme-modern.sky-theme-brand-base.sky-theme-mode-dark.local-preview-layering {
+	--_layer-0-bg: var(--sky-color-background-page);
+	--_layer-1-bg: var(--sky-color-background-container-base);
+	--_layer-2-bg: var(--sky-color-background-container-backdrop, var(--bb-color-gray-900));
+	--_layer-3-bg: var(--sky-color-background-container-menu, var(--bb-color-gray-850));
+	min-height: 100vh;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)